freeipa-api/src/main/java/com/sequenceiq/service/api/doc/ModelDescriptions.java

```diff
@@ -6,7 +6,8 @@
     public static final String NAME = "name of the resource";
     public static final String DESCRIPTION = "description of the resource";
     public static final String ENVIRONMENT_NAME = "The name of the environment";
-    public static final String ENVIRONMENT_CRN = "The crn of the environment";
+    public static final String ENVIRONMENT_CRN = "CRN of the environment";
+    public static final String USER_CRN = "CRN of a user. It is optional as it will be available in \"x-cdp-actor-crn\" header in the request.";
```

Comment:
you say here it's optional, while in the request it has NotNull annotation. I would omit this field if this is not really necessary

Reply:
It is addressed in https://github.com/hortonworks/cloudbreak/pull/5441